src/events/keyboard.js

```diff
@@ -325,27 +333,16 @@ p5.prototype._onblur = function(e) {
  *
  * function setup() {
  *   createCanvas(512, 512);
+ *   fill(255, 0, 0);
  * }
  *
  * function draw() {
- *   if (keyIsDown(LEFT_ARROW)) {
- *     x -= 5;
- *   }
- *
- *   if (keyIsDown(RIGHT_ARROW)) {
- *     x += 5;
- *   }
- *
- *   if (keyIsDown(UP_ARROW)) {
- *     y -= 5;
- *   }
- *
- *   if (keyIsDown(DOWN_ARROW)) {
- *     y += 5;
- *   }
+ *   if (keyIsDown(LEFT_ARROW)) x -= 5;
```

Comment:
I'm worried about removing the `{` and `}` from these if statements since it may confuse beginners who aren't familiar with JS syntax. ("Are the braces required or not?") Does anyone else have thoughts on this? I don't have a strong opinion.

Reply:
I think it is good to keep the braces for clarity.